app/assets/javascripts/float-kit/addon/components/d-menu.gjs

```diff
@@ -77,6 +77,7 @@ export default class DMenu extends Component {
         "fk-d-menu__trigger"
         (if this.menuInstance.expanded "-expanded")
         (concat this.options.identifier "-trigger")
+        this.options.extraClassName
```

Comment:
Need this to give the dmenu trigger button another class that's not the identifier, since i want the identifier to be unique with type/id

Reply:
I removed the need for this